docs/src/pages/[platform]/theming/style-props/index.page.mdx

```diff
@@ -0,0 +1,175 @@
+---
+title: Style Props
+metaDescription: 'CSS Style Props that are available on every Amplify UI component'
+supportedFrameworks: react
+---
+
+import { Example, ExampleCode } from '@/components/Example';
+import { ComponentPropsToStylePropsMap, Card } from '@aws-amplify/ui-react';
+import {
+  BackgroundStyles,
+  BackgroundThemeTokenExample,
+  BorderStyles,
+  BorderThemeTokenExample,
+  ColorStyles,
+  ColorThemeTokenExample,
+  FlexStyles,
+  FlexStylePropExample,
+  GridStyles,
+  GridStylePropExample,
+  MarginAndPaddingStyles,
+  MarginAndPaddingThemeTokenExample,
+  PositionStyles,
+  PositionThemeTokenExample,
+  ShadowStyles,
+  ShadowThemeTokenExample,
+  SizeStyles,
+  SizeThemeTokenExample,
+  TypographyStyles,
+  TypographyThemeTokenExample,
+  OtherStyles,
+} from './examples';
+
+These style props are available on Amplify UI primitive components and can be used to modify the corresponding styles. This allows users to quickly make style adjustments without having to go through CSS or add a theme object. Style props override any conflicting CSS style sheet or theme styling.
+
+There are three ways of passing style props to a component:
+1. Pass any valid CSS property directly (e.g., backgroundColor="red")
+- This is useful for rapid prototyping or styling a component which differs from the main theme
+2. Use a design token from the Theme object by destructuring tokens from the useTheme hook `(e.g., backgroundColor={tokens.colors.red[40]}`
+- This method should be used if using Amplify UI's theming capabilities and will update with changes to the theme, such as setting dark mode overrides.
+3. Use the shorthand syntax to reference design tokens without using the useTheme hook (e.g., backgroundColor="red.40")
+- This is a shorthand for method 2. It can be used for convenience when interacting with style props, however, not all design tokens have a shorthand mapping.
```

Comment:
"This is a convenient shorthand for method 2. However, not all design tokens have a shorthand mapping."